hoomd/md/methods/methods.py

```diff
@@ -30,7 +32,104 @@ def _detach_hook(self):
         self._simulation.state.update_group_dof()
 
 
-class NVT(Method):
+class Thermostat(_HOOMDBaseObject):
+    def __init__(self, kT):
+        param_dict = ParameterDict(kT=Variant)
+        param_dict["kT"] = kT
+        self._param_dict.update(param_dict)
+        self._cpp_obj = None
+
+    @hoomd.logging.log(requires_run=True)
+    def thermostat_energy(self):
+        """Energy the thermostat contributes to the Hamiltonian \
+        :math:`[\\mathrm{energy}]`."""
+        return self._cpp_obj.getThermostatEnergy(self._simulation.timestep)
+
+    def _attach_hook(self):
+        pass
+
+
+class ConstantEnergy(Thermostat):
+    def __init__(self):
+        super().__init__(1.0)
```

Comment:
Remove `kT` from the `param_dict` here so the `ConstantEnergy` class does not have that attribute.